src/components/Main/Main.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect, useCallback } from "react";
+import Style from "./Main.module.css";
+import { Row, Col, DatePicker, Image, Spin, notification } from "antd";
+import moment from "moment";
+
+const Main = () => {
+  const [loading, setLoading] = useState(false);
+  const [date, setDate] = useState(null);
+  const [image, setImage] = useState(null);
+  const [title, setTitle] = useState("");
+  const [infoImage, setInfoImage] = useState(null);
+  const { REACT_APP_API_KEY_NASA_POD, REACT_APP_API_URL } = process.env;
+
+  const onChangeDate = useCallback(
+    async (datemoment, datestring) => {
+      setLoading(true);
+      setDate(datemoment);
+      try {
+        const response = await fetch(
+          `${REACT_APP_API_URL}?api_key=${REACT_APP_API_KEY_NASA_POD}&date=${datestring}`
+        );
+        const POD = await response.json();
+        updateStates(POD);
+        if (POD.msg) {
+          sendNotificationError(POD);
+        }
+      } catch (error) {
+        sendNotificationError("There was an error, please try again.");
+      }
+      setLoading(false);
+    },
+    [REACT_APP_API_KEY_NASA_POD, REACT_APP_API_URL]
+  );
+
+  const sendNotificationError = ({ msg = "" }) => {
+    notification["error"]({
+      message: "Error",
+      description: msg,
```

Comment:
We should disable auto close notifications to avoid inestable tests incase timeouts end before test suites and maybe close them through afterEach();
```suggestion
      description: msg,
      duration: 0,
```